src/main/java/htsjdk/samtools/util/SortingCollection.java

```diff
@@ -141,7 +143,8 @@
      * @param tmpDir          Where to write files of records that will not fit in RAM
      */
     private SortingCollection(final Class<T> componentType, final SortingCollection.Codec<T> codec,
-                              final Comparator<T> comparator, final int maxRecordsInRam, final Path... tmpDir) {
+                              final Comparator<T> comparator, final int maxRecordsInRam,
+                              final boolean sampleRecordSize, final Path... tmpDir) {
```

Comment:
Can you add javadoc for `sampleRecordSize`?

Reply:
Sure. I added it to the public methods.. but I can add it here too no problem.